common.go

```diff
@@ -37,7 +39,8 @@ func getMetrics(cmd *cobra.Command, metricsProfile string) []string {
 	case Both:
 		metricsProfiles = []string{"metrics-report.yml", metricsProfile}
 	}
-	return metricsProfiles
+	os.Setenv("TS_METRICS", metricsProfile)
+	os.Setenv("METRICS", strings.Join(metricsProfiles, ","))
```

Comment:
Raising a point I think was lost in the other thread .. Do we need this separation of `TS_METRICS` and `METRICS` ? Can we not simply default to the same logic, which can be toggled via CLI arguments?

In all the configs and here, I would remove this split logic and reduce to `METRICS` only.

Reply:
Yes, make sense. Without code actually splitting the indexing logic, we do not need a split in metrics. Keeping `METRICS` only as suggested.

Reply:
Thank you! :)